index.js

```diff
@@ -325,24 +326,63 @@ if(!helpers.isTestEnvironment()){
 }
 
 // Handler for income SmartThings POST requests
-app.post('/api/st', function(req, res, next) {    // eslint-disable-line no-unused-vars -- next might be used in the future
-    smartapp.handleHttpCallback(req, res);
-});
+app.post('/api/st', Validator.body(['lifecycle']).exists(), function(req, res, next) {    // eslint-disable-line no-unused-vars -- next might be used in the future
```

Comment:
Could you please add some tests to show that these Validators work as expected?

Reply:
I have a question about this in the dev channel because I am not sure what the header authorisation token for smartapp is or how to find it, it gets to that function though meaning it passes the validator which is also in that message so I'm not sure if that would be testing enough to check?

https://bravely.slack.com/archives/CE32NB85N/p1610578347334700

Reply:
Can you stub out the smartapp function that looks for the token?

Reply:
Is it just this function you were wanting added tests for? If the express thing doesn't work the existing tests fail is how it's otherwise working

Reply:
I have added tests for this but Samsung itself throws an error on top of returning a 401 (which the test does look for and gets), is that a problem? I'm not sure how to get around it

Reply:
Looks like you figured it out. Thanks for adding the tests. They look good.